From 5fe42b913368b988c1b6e3dec2ce8df03bc87bc9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Norbert=20Kami=C5=84ski?= <norbert.kaminski@3mdeb.com>
Date: Fri, 11 Dec 2020 17:52:16 +0100
Subject: [PATCH] Makefile: Allow to overwrite include path
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Norbert Kamiński <norbert.kaminski@3mdeb.com>
---
 Makefile.ossl | 2 +-
 Makefile.psa  | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.ossl b/Makefile.ossl
index 2596ce8..5b7e5e5 100644
--- a/Makefile.ossl
+++ b/Makefile.ossl
@@ -40,7 +40,7 @@ QCBOR_LIB=-lqcbor
 # These two are for reference to OpenSSL that has been installed in
 # /usr/local/ or in some system location.
 CRYPTO_LIB=-l crypto
-CRYPTO_INC=-I /usr/local/include
+CRYPTO_INC?=-I /usr/local/include
 
 CRYPTO_CONFIG_OPTS=-DT_COSE_USE_OPENSSL_CRYPTO
 CRYPTO_OBJ=crypto_adapters/t_cose_openssl_crypto.o
diff --git a/Makefile.psa b/Makefile.psa
index 709851c..bbcba11 100644
--- a/Makefile.psa
+++ b/Makefile.psa
@@ -27,7 +27,7 @@
 # This is for reference to QCBOR that has been installed in
 # /usr/local/ or in some system location. This will typically
 # use dynamic linking if there is a libqcbor.so
-QCBOR_INC=-I /usr/local/include
+QCBOR_INC?=-I /usr/local/include
 QCBOR_LIB=-lqcbor
 
 
@@ -42,7 +42,7 @@ QCBOR_LIB=-lqcbor
 # These two are for reference to MBed Crypto that has been installed in
 # /usr/local/ or in some system location.
 CRYPTO_LIB=-l mbedcrypto
-CRYPTO_INC=-I /usr/local/include
+CRYPTO_INC?=-I /usr/local/include
 
 CRYPTO_CONFIG_OPTS=-DT_COSE_USE_PSA_CRYPTO
 CRYPTO_OBJ=crypto_adapters/t_cose_psa_crypto.o 
-- 
2.25.1

